src/openrct2/peep/Peep.h

```diff
@@ -632,36 +632,36 @@ struct Peep : SpriteBase
     money16 paid_on_drink;
     uint8_t ride_types_been_on[16];
     uint32_t item_extra_flags;
-    uint8_t photo2_ride_ref;
-    uint8_t photo3_ride_ref;
-    uint8_t photo4_ride_ref;
-    uint8_t current_ride;
-    StationIndex current_ride_station;
-    uint8_t current_train;
+    uint8_t Photo2RideRef;
+    uint8_t Photo3RideRef;
+    uint8_t Photo4RideRef;
+    ride_id_t CurrentRide;
```

Comment:
You can't change the field size without recreating all the replays.

Reply:
Argh, right, let me revert it (ended up reverting the wrong patch